app/api/v2/handlers/fact_source_api.py

```diff
@@ -22,44 +22,77 @@ def add_routes(self, app: web.Application):
         router.add_put('/sources/{id}', self.create_or_update_source)
         router.add_delete('/sources/{id}', self.delete_source)
 
-    @aiohttp_apispec.docs(tags=['sources'])
+    @aiohttp_apispec.docs(tags=['sources'],
+                          summary='Retrieve all fact sources.',
+                          description='Returns a list of all fact sources, including custom-created ones.')
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
-    @aiohttp_apispec.response_schema(SourceSchema(many=True, partial=True))
+    @aiohttp_apispec.response_schema(SourceSchema(many=True, partial=True), description='Returns a list of all Sources dumped in SourceSchema format.')
     async def get_fact_sources(self, request: web.Request):
         sources = await self.get_all_objects(request)
         return web.json_response(sources)
 
-    @aiohttp_apispec.docs(tags=['sources'])
+    @aiohttp_apispec.docs(tags=['sources'],
+                          summary='Retrieve a fact source by its id.',
+                          description='Returns a fact source, given a source id.',
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'id',
+                              'description': 'The id of the fact source',
+                              'schema': {'type': 'string'},
+                              'required': 'true'
+                            }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
-    @aiohttp_apispec.response_schema(SourceSchema(partial=True))
+    @aiohttp_apispec.response_schema(SourceSchema(partial=True), description='Returns a single Source dumped in SourceSchema format.')
     async def get_fact_source_by_id(self, request: web.Request):
         source = await self.get_object(request)
         return web.json_response(source)
 
+    @aiohttp_apispec.docs(tags=['sources'],
+                              summary='Create a fact source.',
+                              description='Returns a new fact source, given a name.',
+                              parameters=[{
+                                  'in': 'path',
+                                  'name': 'name',
+                                  'description': 'The name of the new fact source',
+                                  'schema': {'type': 'string'},
+                                  'required': 'true'
+                                }])
```

Comment:
For POST requests we don't have any URL parameters, so the `parameters` can be removed and description can be updated to just be something like 
'Create a new Fact Source using the format provided in the '`SourceSchema`'